.gitignore

```diff
@@ -122,3 +122,5 @@ bench/Micro.Benchmark/BenchmarkDotNet.Artifacts/
 /src/Raven.Pal/osxcross
 /src/Raven.Pal/*.nativecodeanalysis.xml
 /src/Raven.Pal/src/posix/*.o
+/src/Raven.Pal/Debug
+/src/Raven.Pal/build-all-posix.log
```

Comment:
Is this needed?

Reply:
No. I will delete those, it should be in .gitignore